.github/workflows/ClimaLandSimulations.yml

```diff
@@ -0,0 +1,34 @@
+name: ClimaLandSimulations CI
+on:
+  pull_request:
+
+jobs:
+  lib-climalandsimulations:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 45
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: '1.10'
+      - uses: actions/cache@v3
+        env:
+          cache-name: cache-artifacts

```

Comment:
There's a cache action specific for julia, see it here: https://github.com/CliMA/ClimaDiagnostics.jl/blob/main/.github/workflows/CI.yml

It also requires the permissions block:
```
# Needed to allow julia-actions/cache to delete old caches that it has created
permissions:
  actions: write
  contents: read
```